kaldb/src/test/java/com/slack/kaldb/testlib/KaldbConfigUtil.java

```diff
@@ -88,4 +70,26 @@ public static void initEmptyIndexerConfig() throws InvalidProtocolBufferExceptio
         .setMetadataStoreConfig(metadataStoreConfig)
         .build();
   }
+
+  public static int TEST_INDEXER_PORT = 10000;
+
+  public static KaldbConfigs.IndexerConfig makeIndexerConfig() {
+    return makeIndexerConfig(TEST_INDEXER_PORT);
+  }
+
+  public static KaldbConfigs.IndexerConfig makeIndexerConfig(int indexerPort) {
```

Comment:
It would be better if we called these `get...Config`, as that more clearly indicates they do not mutate state. `Make` is significantly more ambiguous, and requires reviewing the source to determine if it does indeed deal with state or is static.

Reply:
In general, get is used for a field access and using make for static methods. Also, the name is more consistent with existing convention, so left it as is.

Reply:
This is somewhat of an anti-pattern, as `get` is expected to be used for methods that return a "non-boolean function or attribute of the object on which they're invoked." Using `make` is fine, but `get` would be a more appropriate use.